tests/acceptance/smoke-test.js

```diff
@@ -67,6 +70,36 @@ test('reordering with mouse events', function(assert) {
     assert.equal(verticalContents(), 'Uno Dos Tres Cuatro Cinco');
     assert.equal(horizontalContents(), 'Uno Dos Tres Cuatro Cinco');
     assert.equal(tableContents(), 'Uno Dos Tres Cuatro Cinco');
+    assert.equal(scrollableContents(), 'Uno Dos Tres Cuatro Cinco');
+  });
+
+  let itemHeight = () => {
+    let item = findWithAssert('.scrollable-demo .sortable-item');
+    return item.outerHeight() + parseInt(item.css('margin-top'));
+  };
+
+  drag(
+    'mouse',
+    '.scrollable-demo .handle[data-item=Uno]',
+    () => {
+      return { y: itemHeight() + 1 };
+    },
+    {
+      dragend: function() {
```

Comment:
At first I tried adding these sort of callbacks to the `reorder` helper, but that didn't really work because this test is has to rely on the offset to make sense, and `reorder` works by targeting the elements directly.